__tests__/app.test.js

```diff
@@ -156,14 +156,44 @@ describe("GET /api/users", () => {
       .expect(200)
       .then(({ body: { users } }) => {
         expect(users).toHaveLength(4);
-        expect(users).toBeInstanceOf(Object)
+        expect(users).toBeInstanceOf(Object);
         expect(users).toEqual([
           { username: "butter_bridge" },
           { username: "icellusedkars" },
           { username: "rogersop" },
-          { username: "lurker" }
-        
+          { username: "lurker" },
```

Comment:
I would still recommend the use of `expect.ObjectContaining` here so you are not committing yourself to a sort order/ properties inside each object 👍